sklearn/cluster/tests/test_optics.py

```diff
@@ -29,14 +32,125 @@
 X = np.vstack((C1, C2, C3, C4, C5, C6))
 
 
+@pytest.mark.parametrize(
+    'data',
```

Comment:
why is this not `r_plot,end,index,size`?

Reply:
cause I didn't think of it :P